### ISSUE TITLE
Winget completion throws error

### ISSUE THREAD

**Issue by HeyItsJono** [vladimir-kotikov/clink-completions#169]
Sorry for the double report but unfortunately `winget` completion also throws an error currently:

    loading completion script failed:
    ...ng\Cmder\config\clink-completions\completions\winget.lua:497: attempt to index local 'arghelper' (a boolean value)
    stack traceback:
            ...ng\Cmder\config\clink-completions\completions\winget.lua:497: in main chunk
            [C]: in function 'dofile'
            {arguments.lua}: in function <{arguments.lua}:2013>
            [C]: in function 'xpcall'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function 'classify'
            {classifier.lua}: in function <{classifier.lua}:63>
            [C]: in function 'xpcall'
            {classifier.lua}: in function <{classifier.lua}:62>
    t
    loading completion script failed:
    ...ng\Cmder\config\clink-completions\completions\winget.lua:497: attempt to index local 'arghelper' (a boolean value)
    stack traceback:
            ...ng\Cmder\config\clink-completions\completions\winget.lua:497: in main chunk
            [C]: in function 'dofile'
            {arguments.lua}: in function <{arguments.lua}:2013>
            [C]: in function 'xpcall'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function '?'
            {arguments.lua}: in function 'classify'
            {classifier.lua}: in function <{classifier.lua}:63>
            [C]: in function 'xpcall'
            {classifier.lua}: in function <{classifier.lua}:62>

This is using the latest version of Clink (v1.4.23.5fc3fc) and a fresh pull from this repo.

**Comment by HeyItsJono** [comment]
I think I've found the issue, it's with the separate [clink-gizmos](https://github.com/chrisant996/clink-gizmos) repo, which I also have installed. The `.init.lua` script registers its own `modules` folder which contains `arghelper.lua`, which presumably is an older version compared to the one shipped with clink-completions. As a workaround I've simply edited clink-gizmos' `.init.lua` file to point to clink-completions' `modules` folder instead of its own:

`package.path = debug.getinfo(1, "S").source:match[[^@?(.*[\/])[^\/]-$]] .."modules/?.lua;".. package.path`

becomes

`package.path = debug.getinfo(1, "S").source:match[[^@?(.*[\/])[^\/]-$]] .."../clink-completions/modules/?.lua;".. package.path`

And in doing this I don't get the bug any longer.

**Comment by chrisant996** [comment]
> I think I've found the issue, it's with the separate [clink-gizmos](https://github.com/chrisant996/clink-gizmos) repo, which I also have installed. The `.init.lua` script registers its own `modules` folder which contains `arghelper.lua`, 

Aha.  But **no**, look at the clink-gizmos repo on github...  Its `modules\` directory does _**not**_ contain arghelper.lua or clink_version.lua files.  The `arghelper.lua` script was removed from there in August 2022.  And it has never contained a `clink_version.lua` script.

I think something is wrong with how the directories were populated on your computer, or with how/when the git repos are being updated.

**Comment by chrisant996** [comment]
> As a workaround I've simply edited clink-gizmos' `.init.lua` file to point to clink-completions' `modules` folder instead of its own:
> 
> `package.path = debug.getinfo(1, "S").source:match[[^@?(.*[\/])[^\/]-$]] .."modules/?.lua;".. package.path`
> 
> becomes
> 
> `package.path = debug.getinfo(1, "S").source:match[[^@?(.*[\/])[^\/]-$]] .."../clink-completions/modules/?.lua;".. package.path`

No, that is working around a problem where your `clink-gizmos\modules\` directory _**contains the wrong files**_.

Instead of changing the program code, fix the wrong file system contents.

**Comment by HeyItsJono** [comment]
Hey sorry for the slow reply; different timezones. You have eagle-eyes I swear, I didn't even notice that `.` was in the list of script directories, so I've uninstalled that. You are also absolutely correct the issue was I had an old version of clink-gizmos which had an `arghelper.lua` script in the modules folder. I have updated my clink-gizmos folder by doing a fresh pull from the current repo and the issue is fixed now.

There was never  a `clink-version.lua` in the clink-gizmos module folder but there was one in the clink-completions module folder shipped in Cmder's vendor folder, which I have since deleted as per the other issue opened.
